examples/redis-operator.md

```diff
@@ -0,0 +1,109 @@
+# Redis-pmem operator
+This a complete example to deploy a Redis cluster through the [redis-operator](https://github.com/spotahome/redis-operator) using QEMU-emulated persistent memory devices.
+
+## PMEM-CSI installation
+The steps to install PMEM-CSI within a K8s cluster with one master and three workers nodes using a fix version of [Clear Linux OS](https://clearlinux.org/) for the nodes, so the following steps can be reproduced, are listed below:
+1. Clone this project into `$GOPATH/src/github.com/intel`
+2. Build PMEM-CSI:
+    ```sh
+     $ cd $GOPATH/src/github.com/intel/pmem-CSI
+     $ make push-images
+    ```
+    **Note**: by default the build images are going to be pushed into a local [Docker registry](https://docs.docker.com/registry/deploying/).
+
+3. Start the K8s cluster
+    ```sh
+    $ TEST_CLEAR_LINUX_VERSION=29820 make start
+    ```
+    **WARNING**: you may run into a SSL problem when executing this command, which can be avoided by disabling the signed files test checks, so the final command to be executed will be: `TEST_CLEAR_LINUX_VERSION=29820 TEST_CHECK_SIGNED_FILES=false make start`, be aware that **this is not a secure step**.
```

Comment:
Replace with this text: 
**WARNING**: You may run into an SSL problem when executing this command. You can avoid the SSL error by disabling test checks for signed files, however, be aware that this is not a secure step. Use this command to disable checking for signed files: 
`TEST_CLEAR_LINUX_VERSION=29820 TEST_CHECK_SIGNED_FILES=false make start`